Fix event documentation example (#28402)
home-assistant/home-assistant.github.io: source/_integrations/event.markdown

This allows you, for example, to trigger a different action when the button on a remote control is pressed once or twice, if your remote control is capable of emitting these different types of events.

> When combining that with the [choose action](/docs/scripts/#choose-a-group-of-actions) script, you can assign multiple different actions to a single event entity. In the following example, pressing the button on the remote short or long will trigger a different scene:

Reviewer: ```suggestion
When combining that with the [choose action](/docs/scripts/#choose-a-group-of-actions) script, you can assign multiple different actions to a single event entity. In the following example, a different scene will be triggered depending on whether you press or press and hold the button:
```
The MS Style Guide seems to use _press and hold_ / _select and hold_
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/p/press-and-hold

also google:
https://support.google.com/accessibility/android/answer/7650693?hl=en

Alternatively, Apple use the term long-press:
https://developer.apple.com/documentation/uikit/touches_presses_and_gestures/handling_uikit_gestures/handling_long-press_gestures

Author: Not in this case, as it is a physical remote, and not an UI button.

As per Microsoft:

![CleanShot 2023-08-02 at 09 44 16](https://github.com/home-assistant/home-assistant.io/assets/195327/bcb47acb-f849-43ee-8af5-aa7a7d9785a9)

This is a hardware button, so matches the guides.

Reviewer: The style guide refers to a physical button.
![image](https://github.com/home-assistant/home-assistant.io/assets/38767475/ec5963da-702d-492c-883f-292774998e32)

  - alias: "Choose an action based on the type of event"
    choose:
      - conditions:
        - alias: "Normal evening scene if the button was pressed shortly"
          condition: state
          entity_id: event.hue_remote_control_on_button
          attribute: "event_type"
          state: "short_release"
        sequence:
          - scene: scene.living_room_evening
      - conditions:

>         - alias: "Scene for watching a movie if the button was pressed long"

Reviewer: ```suggestion
        - alias: "Scene for watching a movie if the button was pressed and held"
```

Author: This is matching the Hue wording, that is used in the example.
